app.py

```diff
@@ -130,3 +131,37 @@ def index():
         logger.critical("Caught unhandled exception.", exc_info=True)
 
     return ""
+
+
+@app.schedule("cron(5 0 * * *)")
+def check_user_locks():
+    api_url = "http://localhost:8010/users"
```

Comment:
Does this really work? I thought it would need to go via the API gateway

Reply:
Locally, yes :) But not in prod, need to use the backend url here :)